kempner_computing_handbook/s4_neuro_ai_workflows/README.md

```diff
@@ -1,2 +1,10 @@
 # Spike Sorting
-TBD
\ No newline at end of file
+
+Spike sorting is a crucial step in analyzing electrophysiological data from the brain. In collaboration with the Allen Institute for AI, the Kempner Research Engineering team developed these pipelines to support researchers on the FASRC and HMS RC computing clusters.
+
+Please refer Cluster specific documentation. 
```

Comment:
Thanks @dmbala. I would suggest adding hyperlink to the followings:
- Allen Institute for AI
- FAS Research Computing (at Harvard University)
- Harvard Medical School Research Computing. 

And drop the following bullet points. 
I am open to discuss it further.

Reply:
Good points. Now I added the suggested changes. 

added links to Allen, FASRC, and HMS-RC
removed bullet points from the list